target/linux/ramips/image/mt7621.mk

```diff
@@ -171,6 +171,27 @@ define Device/ampedwireless_ally-00x19k
 endef
 TARGET_DEVICES += ampedwireless_ally-00x19k
 
+define Device/arcadyan_wg630223
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := WG630223
+  IMAGE_SIZE := 24576k
+  KERNEL_SIZE := 4352k
+  UBINIZE_OPTS := -E 5
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | \
+	uImage none | arcadyan-trx 0x30524448 | pad-to $$(KERNEL_SIZE)
```

Comment:
u-boot 2018.09 supports fit image. Maybe you can try it if you have time. 0f068e7c4a83bcbf20c4e52a5f8a3f1fe2af2246 13308161788c98ae6cd48c22b13339fdb8c77130

Reply:
I'm actually running a FIT image on my own device, will push fixes later